web/schemas/ID1A3.json

```diff
@@ -64,12 +64,20 @@
     },
     {
         "key": "StaffScientist",
-        "type": "string",
+        "type": "list_str",
         "optional": false,
         "multiple": true,
         "section": "User",
         "description": "List staff scientists",
-        "placeholder": "KoJYP, NygrenKE"
+        "placeholder": ""
+        "value": [
+            "",
```

Comment:
the first value will be used as default on in drop down menu on MetaData service web UI. Do you want it to be empty? Please confirm.

Reply:
Given this is list_str and multiple is set as true, can there by more than one entry for placeholder?

Reply:
I doubt it can be easily done because it will be converted into drop down menu where specific value will be shown. Multiple entries means that we need to introduce notations which will allow that and parse them accordingly. Do you really need it? What I think can be done on web interface is multi-selection option, but it will not be a dropdown menu.

Reply:
The default or placeholder does not necessarily have to have more than one selected, but we still would like users to be able to select more than one. Does that work?

Reply:
Yes, it will work, see attachment. For that you need to keep `"multiple": true` key-value pair in your config section. This is how it will look like on the web
![Screen Shot 2023-08-20 at Aug 20, 1 54 50 PM](https://github.com/vkuznet/ChessDataManagement/assets/125138/f298b9f3-c7cd-4810-9338-602eb396b402)